site/_posts/2017-04-03-Announcing-Fine-Beta-Release.html.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Fine Beta Release
+author: chessbyte
+date: 2017-04-03 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Fine!
+
+Fine is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. In this release, a lot of attention was given to Ansible. However, other providers, REST API, as well as performance also benefit in this release.
+
+* Added **"Ansible Playbook"** to the list of catalog item types.
+* Automate Model: Add Amazon block storage automation models.
+* Adding new report and widgets for Containers and Add option for container performance reports.
+* Add Alert Definitions (MiqAlert) bulk edits support.
+* Middleware: Hawkular Allow adding datawarehouse provider with a port other than 80.
+
+If you want to hear about features, improvements, and fixes directly from the developers, check out the [Last Week in ManageIQ blog posts](http://manageiq.org/blog/tags/LWIMIQ/).
+
+Take a look at the [changelog](https://github.com/ManageIQ/manageiq/blob/fine/CHANGELOG.md/) for more details.
+
+We encourage you to download the Fine Beta Release and try it out!
+
+
+| Appliance | Format | Size |
+| --------- | ------ | ---- |
+| [Docker](https://hub.docker.com/r/manageiq/manageiq/) | docker | 0.5 GB |
+| [Google Compute Engine](http://releases.manageiq.org/manageiq-gce-fine-1-beta1.gz) | gce | 1.1 GB |
```

Comment:
Sorry I missed this earlier, looks like file ext for GCE is tar.gz instead of just gz